packages/itwinui-css/backstop/tests/icon.html

```diff
@@ -105,22 +105,22 @@ <h2>Actionable icons</h2>
     <h2>Icons with status</h2>
     <section id="demo-status">
       <svg-info-circular
-        class="iui-icons-default iui-informational"
+        class="iui-icon-style-m iui-icon-color-status iui-informational"
```

Comment:
For me `iui-icon-color-status` class looks a bit odd as it doesn't do anything. What if we just have status classes only like `iui-icon-status-informational`?

Reply:
I agree, my suggestion shared [here](https://github.com/iTwin/iTwinUI/pull/701#discussion_r932362274).

Reply:
In cases where `@include iui-icon-color-status` is used, wouldn't I then have to `include` every type?

Reply:
Every status will be included together with `@include iui-icon-color-status`